camel/models/model_factory.py

```diff
@@ -25,8 +30,11 @@ class ModelFactory:
     """
 
     @staticmethod
-    def create(model_type: ModelType,
-               model_config_dict: Dict) -> BaseModelBackend:
+    def create(
+        model_type: ModelType,
+        model_config_dict: Dict,
+        model_path: Optional[str] = None,
```

Comment:
Is it possible to not contain `model_path` parameter in this? I think it is only used to create the tokenizer. So can we assign it inside the factory by `model_type` instead of passing it in?

Reply:
The thing is the current `model_type` has only general information of the model but without the specific model name/path (e.g. `lm-sys/vicuna-7b-v1.5`). 
I might get what you mean by modifying `ModelType` to be a data class? Should we implement another class called like `ModelCard`, which has a field being `ModelType` in limited range and can have other information like model path?